uikit/views/src/main/java/com/sygic/maps/uikit/views/common/extensions/ViewExtensions.kt

```diff
@@ -24,7 +24,28 @@
 
 package com.sygic.maps.uikit.views.common.extensions
 
-import android.widget.EditText
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.annotation.ColorRes
+import androidx.appcompat.content.res.AppCompatResources
+import com.sygic.maps.uikit.views.common.utils.TextHolder
 
-fun EditText.showKeyboard() = context?.showKeyboard(this)
-fun EditText.hideKeyboard() = context?.hideKeyboard(this)
+fun View.showKeyboard() = context?.showKeyboard(this)
+fun View.hideKeyboard() = context?.hideKeyboard(this)
+
+fun View.visible(visible: Boolean) {
```

Comment:
dal by som mozno skor
```suggestion
fun View.visible(visible: Boolean, useGone: Boolean = true) {
```